docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md

```diff
@@ -304,6 +304,54 @@ LOG.info("Write result: {}", result);
 
 </div>
 
+<div id="ingester-json-type">
+
+In the [low-level API](#low-level-api),
+you can specify the column type as `DataType.JSON` using the `addField` method to add a JSON column.
+Then, insert JSON data as a string value.
+
+```java
+// Construct the table schema for sensor_readings
+TableSchema sensorReadings = TableSchema.newBuilder("sensor_readings")
+        // The code for creating other columns is omitted
+        // ...
+        // specify the column type as JSON        
+        .addField("attributes", DataType.JSON)
+        .build();
+
+// ...
+// Add JSON data as a string value
+sensorReadings.addRow(<other-column-values>... , "{\"location\":\"factory-1\"}");
```

Comment:
@fengjiachun I consider if we can provide common code for converting a map to a string, the user experience here will be more friendly.

Reply:
https://github.com/GreptimeTeam/greptimedb-ingester-java/issues/54